lib/kafka/consumer_interceptors.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Kafka
+  # Holds a list of Kafka::ConsumerInterceptor
+  # and wraps calls to a chain of custom consumer interceptors.
+  class ConsumerInterceptors
```

Comment:
I don't know how upstream will feel, but for me it feels a bit weird having `ConsumerInterceptor` and `ConsumerInterceptors` be so similarly named.

Something like `InterceptorList` would be more obviously different, I think, but I don't have strong feelings here.

Reply:
I was following the same names from the [KIP-42](https://cwiki.apache.org/confluence/display/KAFKA/KIP-42%3A+Add+Producer+and+Consumer+Interceptors#KIP42:AddProducerandConsumerInterceptors-ProposedChanges), but I think we could use different names if it's clearer